### DISCUSSION TITLE
AtomicInteger support

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [beehive-lab/TornadoVM#479] [Q&A]
Hi.

I noticed that you have a plugin for AtomicInteger in GPU driver modules, but I did not find any mention of it in the documentation (though I may be a bad reader).
Is this functionality supported?

**Comment by jjfumero** [comment]
Yes, `AtomicIntegers` are supported for the OpenCL backend: 
https://github.com/beehive-lab/TornadoVM/blob/master/tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/atomics/TestAtomics.java
